staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -569,6 +569,27 @@ func isExpiredError(err error) bool {
 	return apierrors.IsResourceExpired(err) || apierrors.IsGone(err)
 }
 
+var tooLargeResourceVersionCauseMsg = "Too large resource version"
+
 func isTooLargeResourceVersionError(err error) bool {
-	return apierrors.HasStatusCause(err, metav1.CauseTypeResourceVersionTooLarge)
+	if apierrors.HasStatusCause(err, metav1.CauseTypeResourceVersionTooLarge) {
+		return true
+	}
+	// In Kubernetes 1.18.5 and earlier, the api server doesn't set status cause
```

Comment:
isn't this just 1.18.0-1.18.5? can we explicitly comment on the version range this is in support of?

Reply:
It covers API servers 1.17.0-1.18.5 (basically from when the "Resource version too large" error was introduced): 
 https://github.com/kubernetes/kubernetes/commit/1406f7adf20a9ceaf5b8270ec9a61dd9e450ae8a#diff-50208dd4bee70b5c6878f68edaac2923R174-R182

Reply:
Changed and called out the explicit versions in the code